### ISSUE TITLE
Ordre différent les sections du formulaire quand on génère un PDF à partir d'un gabarit Word

### ISSUE THREAD

**Issue by florentarneodo** [MTESSDev/FRW#239]
**Votre demande de fonctionnalité est-elle liée à un problème ? Si oui, veuillez décrire.**
Le problème : nos procédures de travail ne suivent pas l'ordre des questions demandé au client.

**Décrivez votre besoin**
On aimerait pouvoir présenter dans un ordre différent les sections du formulaire quand on génère un PDF à partir d'un gabarit Word.

**Décrivez les alternatives que vous avez envisagées**
Refaire un PDF maison, mais c'est un gros formulaire et on perd l'avantage de la flexibilité de FRW.

**Proposition de solution**
L'idée de Dany : pouvoir donner un poids aux sections pour déterminer un ordre de présentation.

**Contexte supplémentaire**
N/A

**Comment by vinnyboii** [comment]
@anisite on avait pensé ajouter une config dans le fichier bind pour ordonnancer les sections du PDF produit par l'étape genererWord.
On doit statuer de quelle façon on permet la config (statique ou en spécifiant un ordre pour une section qui overriderait l'ordre de base)

**Comment by anisite** [comment]
<img width="160" alt="image" src="https://github.com/MTESSDev/FRW/assets/3194288/f64700a5-308e-4337-b5d5-0a79ec9c0c0b">
la fonctionnalité supporterait uniquement les sections numérotées en vert sur l'image, les groupes de sections serais exclus de la portée, est-ce bien cela @florentarneodo  ?

**Comment by florentarneodo** [comment]
Salut @anisite 

J'ai mis en rouge et vert comme toi un exemple concret :

![image](https://github.com/MTESSDev/FRW/assets/140817361/fab1512f-8067-4992-8942-f705db32cb4b)

Ça pourrait donner ça une fois l'ordre changer ? donc sans les titres rouges

![image](https://github.com/MTESSDev/FRW/assets/140817361/25ee648d-640b-4e73-91cf-9e115caa9f6c)

C'est ce que tu essaies de me dire?

**Comment by anisite** [comment]
oui, exactement, c'était là que nous étions dans nos réflexions.

**Comment by florentarneodo** [comment]
On est à l'aise avec ça de notre côté.

**Comment by Ghost3044** [comment]
Mis sur la glace d'ici à ce qu'on obtienne un retour de Retraite Québec
